pom.xml

```diff
@@ -111,6 +106,11 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.zookeeper</groupId>
+      <artifactId>zookeeper</artifactId>
+      <version>${zookeeper.version}</version>
+    </dependency>
```

Comment:
Dependencies should be ordered alphabetically by groupId, then artifactId, then scope. I'm not sure why this one moved.

Reply:
It kept moving when i ran mvn clean compile. I will look at it again.

Reply:
Looking back, I think it moved because it was not ordered alphabetically initially. It is in the correct location now.